crates/bevy_gizmos/src/lib.rs

```diff
@@ -432,6 +433,7 @@ fn extract_gizmo_data(
             },
             (*handle).clone_weak(),
             GizmoMeshConfig::from(config),
+            RenderFlyEntity,
```

Comment:
We discussed this before and it still might be the best option for now, but it feels not great to require users to remember this while extracting.